backend/cognito/template.yml

```
@@ -14,8 +14,13 @@ Parameters:
     Type: String
     Default: ""
     Description: The ARN of the code signing configuration resource to use [Secure Pipeline]
+  ExportPrefix:
+    Type: String
+    Default: ""
+    Description: Used to differentiate stacks in development account
```

```suggestion
  ExportNamePrefix:
    Type: String
    Default: self-service
    AllowedPattern: ^.*[^-]$
    Description: Prefix to use when importing or exporting values from other stacks
```
We might want to use this for any purpose.
Also we can get rid of the condition by requiring the prefix to not end with a hyphen and have a sensible default value that can be used for both exporting and importing values from other stacks.

---

Fancy regex 😄 .